### PR TITLE
Kc 2085 align search pop up usage with design

### PR DESCRIPTION
**Pull request by JoseToitio**

### CODE REVIEW

File: src/components/InputGlobal/index.tsx [hunk 1]
```diff
@@ -48,19 +49,17 @@ const Input: React.FC<InputGlobal> = ({
     if (error) {
       setError(false);
     }
-    if (showTooltip) {
-      setShowTooltip(false);
-    }
-    setSearch(event.target.value);
+    setShowTooltip(true);
+    setSearch(event.target.value.trim().toLowerCase());
```
Comment (lucasportella) [inline src/components/InputGlobal/index.tsx:53]: need to add debounce function

File: src/components/InputGlobal/index.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ const Input: React.FC<InputGlobal> = ({
 }) => {
   const [search, setSearch] = useState('');
   const [error, setError] = useState(false);
+  const [linkPage, setLinkPage] = useState('');
```
Comment (lucasportella) [inline src/components/InputGlobal/index.tsx:21]: is there another user action that needs to be monitored besides the text input or mouse focus in/out of the pop up? if not, just create a variable that stores the data you want to save

File: src/components/InputGlobal/index.tsx [hunk 3]
```diff
@@ -70,11 +69,15 @@ const Input: React.FC<InputGlobal> = ({
 
   const handleTooltipFocus = (e: any) => {
     if (!(e.target.id === 'PrePageTooltip' || e.target.id === 'SearchIcon')) {
-      if (inputRef.current !== null) inputRef.current.value = '';
+      // if (inputRef.current !== null) inputRef.current.value = '';
```
Comment (lucasportella) [inline src/components/InputGlobal/index.tsx:72]: delete comment?

File: src/components/PrePageTooltip/RowSections/index.tsx [hunk 4]
```diff
@@ -346,15 +350,17 @@ export const BlockRowSections = (
...
   if (res.data || res.error === '') {
     const { block } = res.data;
     const date = formatDate(block.timestamp);
+    handleSearch(`/block/${block.nonce}`);
     return [
       {
         element: (
           <TitleWrapper>
-            <TitleSpan>Block Summary</TitleSpan>
+            <TitleSpan>Block Summaryy</TitleSpan>
```
Comment (lucasportella) [inline src/components/PrePageTooltip/RowSections/index.tsx:363]: typo

File: src/components/InputGlobal/index.tsx [hunk 5]
```diff
@@ -70,11 +69,15 @@ const Input: React.FC<InputGlobal> = ({
...
       setShowTooltip(false);
     }
   };
 
+  const handleClick = () => {
+    setShowTooltip(true);
```
Comment (lucasportella) [inline src/components/InputGlobal/index.tsx:78]: should not set tooltip to true just by clicking in input box, makes no sense:
image

**Review by lucasportella — CHANGES_REQUESTED** [review]

File: src/components/InputGlobal/index.tsx [hunk 6]
```diff
@@ -48,19 +48,14 @@ const Input: React.FC<InputGlobal> = ({
...
     }
-    if (showTooltip) {
-      setShowTooltip(false);
-    }
-    setSearch(event.target.value);
-  };
-
-  const handleSearch = () => {
     setShowTooltip(true);
+    setSearch(event.target.value);
```
Comment (lucasportella) [inline src/components/InputGlobal/index.tsx:52]: this does not solve excessive user requests, must add debounce fn

File: src/components/PrePageTooltip/RowSections/index.tsx [hunk 7]
```diff
@@ -49,14 +49,18 @@ const getRedirectButton = (
   </div>
 );
 
+export const setSessionStorage = (link: string): void => {
```
Comment (lucasportella) [inline src/components/PrePageTooltip/RowSections/index.tsx:52]: i'm not very sure if this is really necessary. Let's discuss this in PR time

**Review by lucasportella — CHANGES_REQUESTED** [review]
must discuss problems in PR time

**Review by lucasportella — APPROVED** [review]

**Review by Doug-Fernando — APPROVED** [review]

**Pull request merged**